src/views/shared/Footer.js

```diff
@@ -72,7 +73,10 @@ function (Okta) {
     },
     toggleLinks: function (e) {
       e.preventDefault();
-      this.$('.js-help-links').slideToggle(200);
+
+      this.$('.js-help-links').slideToggle(200, _.bind(function () {
+        this.$('.js-help').attr('aria-expanded', this.$('.js-help-links').is(':visible'));
```

Comment:
Instead of using `tick()` in our tests, could we move this to named function? Then I think we could spy on it and use something like `Expect.waitForSpyCall`